\documentclass[109pt]{article}
\usepackage{array, xcolor, lipsum, bibentry}
\usepackage[margin=1.4cm]{geometry}
\usepackage[T1]{fontenc}
\usepackage{longtable}
\usepackage{enumitem}
\usepackage{hyperref}

\usepackage[utf8]{inputenc}
%\usepackage{nopageno}
\pagenumbering{gobble}

%~ \pagestyle{empty}
%~ \newcommand{\mymail}{qgeissmann@gmail.com}
%\newcommand{\mymail}{q.geissmann@fu-berlin.de}
%\newcommand{\sendTo}{q.geissmann@fu-berlin.de,qgeissmann@gmail.com}

\newcommand{\mymail}{quentin.geissmann13@imperial.ac.uk}
\newcommand{\sendTo}{quentin.geissmann13@imperial.ac.uk,qgeissmann@gmail.com}

\title{\bfseries\Huge Quentin Geissmann}
\author{\href{mailto:\sendTo}{\mymail}}
\date{}
\definecolor{lightgray}{gray}{0.8}
\newcolumntype{L}{>{\raggedleft}p{0.1\textwidth}}
\newcolumntype{R}{p{0.8\textwidth}}
\newcommand\VRule{\color{lightgray}\vrule width 0.5pt}


\newcommand{\customfootnotetext}[2]{{% Group to localize change to footnote
  \renewcommand{\thefootnote}{#1}% Update footnote counter representation
  \footnotetext[0]{#2}}}% Print footnote text


%~ 
%~ \begin{filecontents}{publication.bib}
%~ @article{lamport1986latex,
  %~ title={LaTEX: User's Guide \&amp; Reference Manual},
  %~ author={Lamport, L.},
  %~ year={1986},
  %~ publisher={Addison-Wesley}
%~ }
%~ @book{knuth2006art,
  %~ title={The art of computer programming: Generating all trees: history of combinatorial generation},
  %~ author={Knuth, D.E.},
  %~ volume={4},
  %~ year={2006},
  %~ publisher={addison-Wesley}
%~ }
%~ \end{filecontents}

\begin{document}

\maketitle

\begin{minipage}[ht]{0.48\textwidth}
Date of Birth: 27$^{th}$ December 1986\\
Address: 38 Woodleigh gardens, SW16 2SY, London, UK\\
Nationality: French\\
Webpage: \href{https://quentin.geissmann.net}{https://quentin.geissmann.net}
\end{minipage}
\vspace{1pt}
\section*{\textsc{Research Experience}}
	\begin{longtable}{L!{\VRule}R}
	2014--2018&\emph{PhD student}. Department of Life Sciences, Imperial College London.
	\textbf{High-throughput Acquisition, Analysis and Alteration of Sleep in \emph{Drosophila}} 
	(Dr. G. Gilestro).
	\vspace{2pt}\\
	~&%\emph{Techniques acquired:}
	\begin{itemize}[topsep=\parskip]
		\setlength\itemsep{-.3em}
		\item Statistical analysis and modelling of large time series
		\item Computer-aided design, 3d printing and electronics
		\item Machine learning applied to behaviour analysis
	\end{itemize}
	\vspace{3pt}\\
	%~ =====================================================	
	2010--2013&\emph{Research technician}. 
	Department of Animal and Plant Sciences, Sheffield University.
	\textbf{Stress, Resistance and Evolution of Bacteria Facing the Insect Immune System}
	(Prof. J. Rolff).\\
%	\vspace{0pt}\\
	%	~&%\emph{Techniques acquired:}
		~&\begin{itemize}[topsep=\parskip]
			\setlength\itemsep{-.3em}
			\item Image processing, computer vision
			\item Experimental microbiology and flow cytometry
			\item Bioinformatics
		\end{itemize}
	\vspace{3pt}\\
	%~ =====================================================
	2010 (six~months)&\emph{Master's placement}.
	Global Health Institute, EPFL (Switzerland).
	\textbf{Molecular and Functional Characterisation of the Peptidoglycan Recognition Protein LC (PGRP-LC) in \emph{Drosophila} immunity}
	(Dr. C. Neyen, Prof. B. Lemaitre).
	\vspace{1pt}\\
		~&%\emph{Techniques acquired:}
		\begin{itemize}[topsep=\parskip]
			\setlength\itemsep{-.3em}
			\item Confocal microscopy
			\item Experimental genetics
			\item Molecular biology
		\end{itemize}
	\vspace{3pt}\\
	%~ =====================================================
	2009 (five~months)&\emph{Master's placement}.
	UMR 1272: Insect Physiology, Signalling and
	Communication, INRA Versailles. \textbf{Electrophysiological Study of Olfactory
	Receptor Neurones of Male \emph{Spodoptera litoralis} in Response to a Female
	Pheromone} (Dr. P. Lucas, PI. S. Anton).\\
		~&%\emph{Techniques acquired:}
		\begin{itemize}[topsep=\parskip,after=\vspace{-10pt}]
			\setlength\itemsep{-.3em}
			\item Electrophysiological data analysis
			\item Single sensillum recording
		\end{itemize}
	
	\end{longtable}

%~ <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
\section*{\textsc{Education}}
\begin{longtable}{L!{\VRule}R}
	2013--2014&\emph{MSc}. \textbf{Bioinformatics and Theoretical Systems Biology}, distinction. Imperial College, London.\\
	2008--2010&\emph {MSc}. \textbf{Integrative Biology and Physiology}, equivalent distinction.
	Specialist modules: Molecular phylogenetics'' and ``Mathematical modelling in biology''.
	Universit\'e Pierre et Marie Curie, Paris. 
	\vspace{5pt}\\
	2005--2008&\emph{BSc}. \textbf{Biology of Organisms}, equivalent first.
	Specialist modules:
	``Behavioural biology'', ``Ecological interactions''. Universit\'e de Bourgogne,
	Dijon.\\
\end{longtable}
%~ <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
\newpage{}
\section*{\textsc{Scientific Computing and Programming}\footnote{Most of my contributions are open-source and publicly available (see \href{http://github.com/qgeissmann}{http://github.com/qgeissmann})}}
In addition to my primary interest in biology, I have extensive experience in computer programming and have developed several scientific applications in various languages:
	\begin{longtable}{L!{\VRule}R}
	\texttt{R}&\emph{Highly competent}: base functions, statistics, algebra, data visualisation and package development.\\
	\texttt{python}&\emph{Highly competent}: scientific computing, package development and web applications.\\
	\texttt{C/C++}&\emph{Highly competent}: OpenCV (image processing \& machine learning),	OpenMP and standard library.\\
	System&\emph{Highly competent}: GNU/Linux.\\
	Web&\emph{Competent}: javascript and HTML/CSS.\\
\end{longtable}
%~ <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<

%	2013-2014&\emph {MSc: ``Bioinformatics and Theoretical Systems Biology''}, distinction. Imperial College, London.\\


\section*{\textsc{Teaching, Supervision and Outreach}}
\begin{longtable}{L!{\VRule}R}
	2017--2018&\emph{Statistics in \texttt{R}} to undergraduate students, teaching assistant, 12h/year.\\
	2017&Public engagement at Imperial College festival: interactive presentation of ethomics, 2h.\\
	2016--2017& Lecture seminar: ``High-throughput analysis of sleep behaviour'' for the Applied Biosciences and Biotechnology MSc, 2h/year.\\
	2014--2017&\emph{\texttt{Python} programming} for the Bioinformatics and Theoretical Systems Biology MSc, teaching assistant, 12h/year.\\
	2014--2018&Supervision of masters and undergraduate students, on average three students per year.\\
	2013&\emph{\texttt{Unix} tools for biologists}, at Next Generation Sequencing workshop, Sheffield University, 3h.\\
\end{longtable}

\section*{\textsc{Publications}\footnote{Detailed list on my webpage (\href{https://quentin.geissmann.net\#publications}{https://quentin.geissmann.net\#publications})}}
\begin{longtable}{L!{\VRule}R}
	2018&\textbf{Q. Geissmann}, L. García Rodriguez, E. J. Beckwith, G. F. Gilestro. Rethomics: an R framework to analyse high-throughput behavioural data. \emph{bioRxiv preprint}.\\
	2017&\textbf{Q. Geissmann}, L. García Rodriguez, E. J. Beckwith, A. S. French, A. R. Jamasb, and G. F. Gilestro. Ethoscopes: An open platform for high-throughput ethomics. \emph{PLoS Biology}.\\
	2017&E. J. Beckwith, \textbf{Q. Geissmann}, A. S. French, and G. F. Gilestro. Regulation of sleep homeostasis by sexual arousal. \emph{eLife}.\\
	2016&S. Fan$^*$, \textbf{Q. Geissmann}$^*$, E. Lakatos$^*$, S. Lukauskas$^*$, A. Ale, A. C. Babtie, P. D. W. Kirk, and M. P. H. Stumpf. MEANS: python package for Moment Expansion Approximation, iNference and Simulation.  \emph{Bioinformatics}.\\
	2014&L. Duvaux, \textbf{Q. Geissmann}, K. Gharbi, J.-J. Zhou, J. Ferrari, C. M. Smadja, and R. K. Butlin. Dynamics of Copy Number Variation in Host Races of the Pea Aphid.  \emph{Mol Biol Evol}.\\
	2013&\textbf{Q. Geissmann}. OpenCFU, a New Free and Open-Source Software to Count Cell Colonies and Other Circular Objects. \emph{PLoS ONE}.\\ %\{http://www.plosone.org/article/info\%3Adoi\%2F10.1371\%2Fjournal.pone.0054072}{doi:10.1371/journal.pone.0054072}.
\end{longtable}


\customfootnotetext{$^*$}{Co-first authorship}

\section*{\textsc{Significant Posters and Presentations}}
\begin{longtable}{L!{\VRule}R}
	2018&Invited speaker: How much sleep does a fly \emph{really} need? \emph{Life Sciences Departmental Seminar, Imperial College London}.\\
	2017&Poster: \textbf{Q. Geissmann}, L. García Rodriguez, E. J. Beckwith, and G. F. Gilestro. Is sleep deprivation really lethal to flies? \emph{European Drosophila Research Conference, London}.\\
	2017&Invited speaker: Is sleep deprivation really lethal to flies? \emph{Champalimaud Centre for the Unknown, Lisboa}.\\
	2017&Poster: \textbf{Q. Geissmann}, L. García Rodriguez, E. J. Beckwith, and G. F. Gilestro. Next generation activity monitoring sheds new light on \emph{Drosophila} sleep. \emph{UK clock club, Oxford}.\\
	2015&Invited speaker: High throughput quantification of sleep in fruit fly. \emph{MRC translational innovation mixers, London}.\\ 
\end{longtable}

%\section*{\textsc{Peer Review Activity}}
%\begin{longtable}{L!{\VRule}R}
%	2013--&\href{http://publons.com/author/1483055/quentin-geissmann}{publon}\\
	%2015&Invited speaker: High throughput quantification of sleep in fruit fly. \emph{MRC translational innovation mixers, London}.\\ 
%\end{longtable}



%\section*{\textsc{Languages}}
%\begin{longtable}{L!{\VRule}R}
%	French&Native speaker\\
%	\emph{English}&\emph{Fluent}\\
%	Spanish&Basic\\
%\end{longtable}


%\section*{\textsc{Other interests}}
%	\begin{itemize}[topsep=\parskip]
%		\setlength\itemsep{-.3em}
%		\item Various outdoors activity including cycling, hiking, running, free diving and gardening.
%		\item Creative activities such as cooking, woodwork and knitting.
%	\end{itemize}
	

\end{document}
